README.md

```diff
@@ -1 +1,23 @@
-# Microverse-portfolio
\ No newline at end of file
+# Setup-and-mobile-version-skeleton
+
+> This is my portfolio Microverse project
+Additional description about the project and its features.
+
+## Built With
+
+- html
+- css
+- github
+
+## Author
+👤 **Author1**
+
+- GitHub: [dadadon](https://github.com/dadadon)
+- Twitter: [Davian Beroni](https://twitter.com/davianberoni)
+- LinkedIn: [Davian Beroni](https://www.linkedin.com/in/davian-beroni-502351b7/)
+
```

Comment:
- Please, kindly update this section of your readme by adding the following headers:

1. Contribution header: Kindly add all the contents in the contributing header, as you've removed some contents from it. You can refer to the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to add the missing contents.
2. Show your support header.

:point_right: The arrangement should follow that used in the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.

Reply:
- Kindly make sure you follow the template stated by microverse [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) structure your readme as stated by the previous reviewer. 
The contribution and the support header seem to be missing in your readme. Kindly add it 👍